### PR TITLE
Create Indexes for All Products

### PR DESCRIPTION
**Pull request by dominicschaff**
All products that provided datasets are now indexed. I have created functions for all the functions, and added them to 2 groups.

The amount of datasets that will be indexed is also customisable by providing the variables INDEX_LIMIT.

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -5,7 +5,8 @@
 ## at 'http://localhost' with password 'secretpassword'
 .PHONY: help setup up down clean
 
-BBOX := 25,20,35,30
+BBOX := 14.6,-36.3,35.9,-20.7
+INDEX_LIMIT := 10
```
Comment (alexgleith) [inline Makefile]: I think this should be bigger, say, `1,000`.
```suggestion
INDEX_LIMIT := 1000
```

**Review by alexgleith — CHANGES_REQUESTED** [review]
Love it! One little suggestion.

File: Makefile (hunk 1 above)
Comment (dominicschaff) [reply Makefile]: I shall test the time this takes, and commit

**Comment by dominicschaff** [comment]
I did also change the default bounding box. So now the bounding box covers south africa

File: Makefile (hunk 1 above)
Comment (dominicschaff) [reply Makefile]: For 10 datasets it takes 5 minutes
For 100 datasets it takes 12 minutes
for 1000 datasets it takes 1.5 hours

**Comment by alexgleith** [comment]
Hmm, 1.5 hours is too long!

Do you think we can do indexing in parallel instead of sequentially?

**Comment by dominicschaff** [comment]
> Hmm, 1.5 hours is too long!
> 
> Do you think we can do indexing in parallel instead of sequentially?

I should be able to. I shall investigate

**Comment by dominicschaff** [comment]
@alexgleith When I ran it now I managed to get it at around 17 minutes.

**Review by alexgleith — APPROVED** [review]

**Pull request merged**